tests/test_ops/test_roi_align.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 import torch
 
+from mmcv.utils import is_cuda, is_mlu
```

Comment:
Are there `is_cuda` and `is_mlu` in `mmcv.utils`?

Reply:
is_cuda and is_mlu would be merged in another pull request (sigmoid_focal_loss).

Reply:
The PR (sigmoid_focal_loss) has been merged so you can use the is_cuda or is_mlu?